Hunting Queries/LAQueryLogs/UserVolumeAnomolies.yaml

```diff
@@ -0,0 +1,34 @@
+id: 8699df3f-f89e-431f-9dea-056c4ce7014a
+name: User returning singificatnly more data than daily average
+description: |
+  'This hunting query looks for users whose total returned data for a day is significantly above their monthly average.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.